sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -41,7 +41,6 @@ interface CreateReceiverOptions {
 }
 
 /**
- * @internal
  * @ignore
```

Comment:
The `@ignore` tag will need to be removed as well for this to show up in documentation.